aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -294,6 +294,12 @@ public class AwsProperties implements Serializable {
 
   public static final String DYNAMODB_TABLE_NAME_DEFAULT = "iceberg";
 
+  public static final String DYNAMODB_V2_SCHEMA_DEFAULT_TABLE_NAME = "iceberg_v2";
```

Comment:
nit: can we name it `DYNAMODB_TABLE_NAME_DEFAULT_V2`? And the default `iceberg_v2` is a bit confusing, can we use `iceberg_dynamodb_v2`?

Reply:
Updated default name